tests/unit_tests/test_source.py

```diff
@@ -96,3 +97,40 @@ def test_source_xml_roundtrip():
     np.testing.assert_allclose(new_src.angle.reference_uvw, src.angle.reference_uvw)
     assert new_src.particle == src.particle
     assert new_src.strength == approx(src.strength)
+
+
+def test_rejection(run_in_tmpdir):
```

Comment:
Sorry I'm late with this comment but do we need this ```run_in_tmpdir``` variable here. It appears that it is not used

Reply:
`run_in_tmpdir` is a pytest fixture that causes the test to be run in a temporary directory. For this test, the reason I added it is because we write XML files and running in a temporary directory means we don't need to worry about cleaning them up afterward